EasyLearn/app/helpers/registrations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module RegistrationsHelper
```

Comment:
If you don't need this file, you can delete it.

Reply:
done.